packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -72,27 +84,45 @@ $tc-toggle-color: #555964 !default;
 	&.muted,
 	&.warning {
 		padding-left: 1px;
-		border-left: 4px solid;
+		border-left: 5px solid;
+		:global(.tc-status-label) {
+			font-weight: 600;
```

Comment:
I don't see anything in the guideline and the result seem bolder. 
Moreover, using 600 (not one of the font-weight that is natively downloaded) cans introduce some anti-aliasing issue in some OS/Browser.
Can you remove that and let the normal (400) weight ?

Reply:
@jsomsanith do we have a *font*, that can be used instead of adding font-weight?

Reply:
In summary, we provide some fonts, and their variations (weight, italic, ...) in some versions only. (Example 400, 700, 900).
If we set a font-weight that is not provided natively, the browser will try to interpolate the font rendering. Depending on the OS and browser, the result can be good (OSX/safari) or bad (Windows/IE).
But either way that's not in the guideline (and does not match the guideline screenshot neither), please remove it.